src/main/java/com/google/codeu/servlets/EventServlet.java

```diff
@@ -0,0 +1,81 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.cloud.vision.v1.EntityAnnotation;
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.Event;
+import com.google.codeu.data.Eventstore;
+import com.google.codeu.data.Message;
+
+/**
+ * Servlet implementation class EventServlet
+ */
+@WebServlet("/EventServlet")
```

Comment:
to match all servlets naming it can be changed to "/event-servlet". Just a little reminder, if you renamed it, remember to change it in the action of the html form as well ;)